src/components/list/list.js

```diff
@@ -69,6 +69,7 @@ class InnerList extends PureComponent {
       >
         <Renderer
           item={item}
+          group={item.isGroup}
```

Comment:
I guess we need test in list.spec.js for this change as well

Reply:
if isGroup already passed in item, why do we need to pass it again in another prop?

Reply:
You have withGroup, isGroup and group, can we consolidate it to one or withGroup and group?